lib/classes/Variables.js

```diff
@@ -32,6 +32,7 @@ class Variables {
 
     // temporally remove variable syntax from service otherwise it'll match
     this.service.defaults.variableSyntax = true;
+    this.service.provider.variableSyntax = true;
```

Comment:
hmmm is it just me, or do I see two identical lines here? am I missing something?!